CMakeLists.txt

```diff
@@ -843,14 +843,14 @@ endif()
 if(IREE_BUILD_PYTHON_BINDINGS)
   # Write out a .env file to make IDEs and developers happy.
   # Yes, we are writing this to the source dir. It is only for IDEs and if
-  # it gets clobbered, it is fine.
+  # it gets clobbered, it is fine (it is also ignored in .gitignore).
   set(_PYTHONPATH_ENV "PYTHONPATH=$<SHELL_PATH:${CMAKE_CURRENT_BINARY_DIR}/compiler/bindings/python;${CMAKE_CURRENT_BINARY_DIR}/runtime/bindings/python>\n")
-  file(GENERATE OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/.env"
-    CONTENT "${_PYTHONPATH_ENV}"
-  )
-  file(GENERATE OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/.env"
-    CONTENT "${_PYTHONPATH_ENV}"
-  )
+  file(GENERATE OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/.env" CONTENT "${_PYTHONPATH_ENV}")
+  file(GENERATE OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/.env" CONTENT "${_PYTHONPATH_ENV}")
+  # Similarly, write out .env.bat for Windows.
+  set(_PYTHONPATH_ENV_BAT "set PYTHONPATH=$<SHELL_PATH:${CMAKE_CURRENT_BINARY_DIR}/compiler/bindings/python;${CMAKE_CURRENT_BINARY_DIR}/runtime/bindings/python>\n")
+  file(GENERATE OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/.env.bat" CONTENT "${_PYTHONPATH_ENV_BAT}")
```

Comment:
We could only write this to `CMAKE_CURRENT_BINARY_DIR` and then drop it from `.gitignore`. The docs that reference it all suggest running from the build directory. It looks like `.env` is used by IDEs in the source dir, which won't be the case for this script.

Before this PR:
```
iree-source/
  .env
iree-build/
  .env
```

After this PR:
```
iree-source/
  .env
  .env.bat  # maybe remove this?
iree-build/
  .env
  .env.bat  # keep this
```

WDYT?

Reply:
Removed iree-source/.env.bat. It's a bit different from .env, and worth keeping the source directory and .gitignore simpler.